### ISSUE TITLE
Tensorflow training stops in iter 6147

### ISSUE THREAD

**Issue by csf0429** [PetroWu/AutoPortraitMatting#28]
Tensorflow training always stops When training iterations reaches 6147.  It seems the loss stills very high, So what should I do to keep model keeps training until convergence？

**Comment by GateStainer** [comment]
I think the loss it reports is just the loss over one batch data set, not the whole data set. So it seems to be high. Computing loss over whole training data may be too costly. The author just trains the model once over whole training set. I suggest to randomly put part of the training data into a set to compute loss?

**Comment by engrchrishenry** [comment]
@csf0429 I am facing the same problem. While running FCN.py the training stops at 6100 step. Did you solve it?

**Comment by csf0429** [comment]
@engrchrishenry The author just trains the model once over whole training set.  The batch data is provided by training list.  You can change the condition in the training loop and set epochs to keep its training.

**Comment by engrchrishenry** [comment]
@csf0429 Sorry for my limited knowledge, but I am very new to deep learning. Could you share the code of how you did it?

**Comment by engrchrishenry** [comment]
@csf0429 Sorry for my limited knowledge, but I am very new to deep learning. Could you share the code of how you did it? Also, how long does it take to train the network? I am using GTX 1080Ti GPU and while running FCN.py it takes about 8 minutes per 100 iterations. Is that normal? If you have successfully trained it then is it possible for you to share the ckpt files? Another issue I have is that FCN_plus.py doesn't run. Its exits without showing any error, not sure whats wrong in there? Sorry again for many questions. Looking forward to your response.
